.github/workflows/nightly-ubuntu-sgx1.yml

```diff
@@ -490,7 +490,7 @@ jobs:
           sudo apt-get remove -y libsgx-ae-epid libsgx-ae-le libsgx-ae-pce libsgx-ae-qe3 \
             libsgx-aesm-launch-plugin libsgx-enclave-common sgx-aesm-service libsgx-urts \
             libsgx-launch libsgx-epid libsgx-quote-ex libsgx-enclave-common-dbgsym \
-            libsgx-ae-qve libsgx-dcap-ql libsgx-dcap-quote-verify || true
+            libsgx-ae-qve libsgx-dcap-ql libsgx-dcap-quote-verify libsgx-dcap-quote-verify-dev || true
```

Comment:
remove `libsgx-dcap-quote-verify-dev` in line 68

Reply:
> remove `libsgx-dcap-quote-verify-dev` in line 68

Done, thanks.